### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by Sh11ntar0**

### CODE REVIEW

File: src/components/Footer.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+.copywrite {
+	text-align: center;
+	margin-top: 4rem;
+
+}
+@media screen and (max-width: 800px) {
+	.footer-links {
+		flex-direction: column;
+		width: 40%;
+		margin-right: 2rem;
```
Comment (DaiSugi01) [inline src/components/Footer.css:40]: delete these ones
and add `margin-bottom: 1rem;`

File: src/components/Footer.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+
+}
+
+.copywrite {
+	text-align: center;
+	margin-top: 4rem;
+
+}
+@media screen and (max-width: 800px) {
```
Comment (DaiSugi01) [inline src/components/Footer.css:36]: 650px

File: src/components/Footer.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	.footer-links {
+		flex-direction: column;
+		width: 40%;
+		margin-right: 2rem;
+	}
```
Comment (DaiSugi01) [inline src/components/Footer.css:41]: .copywrite {
		margin-top: 0rem;
	}

File: src/components/Footer.css [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.footer-links {
+	display: flex;
+	width: 50vw;
+	list-style: none;
+	justify-content: end;
+	/* margin-top: 2rem; */
+	/* margin-right: 2rem; */
+	font-size: 1.5rem;
```
Comment (DaiSugi01) [inline src/components/Footer.css:21]: font-size: 1.3rem;

File: src/components/Footer.css [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+}
+
+.footer-link {
+	margin-left: 2.5rem;
+
```
Comment (DaiSugi01) [inline src/components/Footer.css:27]: add 
`margin-bottom: 1rem;`

File: src/components/Footer.css [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+	margin-top: 4rem;
+
+}
+@media screen and (max-width: 800px) {
+	.footer-links {
+		flex-direction: column;
+		width: 40%;
+		margin-right: 2rem;
+	}
+}
```
Comment (DaiSugi01) [inline src/components/Footer.css:42]: add
 @media screen and (max-width: 800px) {
+       .footer-links {
+               font-size: 1rem;
+       }
+}

**Review by DaiSugi01 — CHANGES_REQUESTED** [review]

File: src/components/Footer.css [hunk 7]
```diff
@@ -0,0 +1,42 @@
+.footer-wrapper{
+	background-color: #515D66;
+	width: 100%;
+	/* height: 200px; */
```
[the remark this answers is not in the document]
Comment (Sh11ntar0) [reply src/components/Footer.css]: fixed

File: src/components/Footer.css [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+}
+.footer-container{
+	display: flex;
+	justify-content: space-between;
+	width: 100%;
+}
+
+.footer-links {
+	display: flex;
+	width: 50vw;
```
[the remark this answers is not in the document]
Comment (Sh11ntar0) [reply src/components/Footer.css]: fixed

File: src/components/Footer.css [hunk 9]
```diff
@@ -0,0 +1,42 @@
+.footer-wrapper{
+	background-color: #515D66;
+	width: 100%;
+	/* height: 200px; */
+	color: #E0E1DD;
+	padding: 2rem;
```
[the remark this answers is not in the document]
Comment (Sh11ntar0) [reply src/components/Footer.css]: fixed

File: src/components/Footer.css (hunk 1 above)
Comment (Sh11ntar0) [reply src/components/Footer.css]: fixed

File: src/components/Footer.css (hunk 2 above)
Comment (Sh11ntar0) [reply src/components/Footer.css]: fixed

File: src/components/Footer.css (hunk 4 above)
Comment (Sh11ntar0) [reply src/components/Footer.css]: fixed

File: src/components/Footer.css (hunk 5 above)
Comment (Sh11ntar0) [reply src/components/Footer.css]: fixed

**Review by DaiSugi01 — APPROVED** [review]

**Pull request merged**